service/src/main/java/bio/terra/policy/service/policy/PolicyUnknown.java

```diff
@@ -72,13 +72,13 @@ public PolicyInput remove(PolicyInput target, PolicyInput removePolicy) {
   }
 
   /**
-   * Unknown policies will be seen as invalid.
+   * Unknown policies will be seen as valid.
```

Comment:
Why this change in behavior?

Reply:
I should have put that in the description :/ This came up because we set these to invalid in an earlier PR, but didn't check that with Product. After more discussion with Product, it makes most sense to require validation for Terra namespace policies, but be flexible with other namespaces.